DTS-STN/Service-Canada-Labs
# Indigenous EI on BDM Project Page and Articles Templates

# [EI on BDM Indigenous research](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=211833)

Created the Indigenous research page and the template to support potential future articles.

## Test Instructions

1. Navigate to EI Indigenous research page from homepage
2. See that page renders properly and is bilingual
